library/string.po

```diff
@@ -974,13 +971,14 @@ msgid ""
 "unless the ``'#'`` option is used."
 msgstr ""
 "Les règles précises sont les suivantes : supposons que le résultat formaté "
```

Comment:
```suggestion
"Les règles précises sont les suivantes : supposons que le résultat formaté "
```

/tmp/padpo_deyl39zb/library/string.po: 961 [     NBSP     ] There should be a non-breakable space before ":": between ###es précises sont les suivantes ### and ###: supposons que le résultat for###